packages/lesswrong/components/posts/SubmitToFrontpageCheckbox.tsx

```diff
@@ -100,7 +100,11 @@ class SubmitToFrontpageCheckbox extends Component<SubmitToFrontpageCheckboxProps
   }
 
   render() {
-    const { classes, label='Moderators may promote to Frontpage', tooltip } = this.props
+    const defaultLabel = forumSelect({
+      EAForum:'Post on Frontpage by default',
```

Comment:
Not to overcomplicate this, but I find this wording quite confusing. From the perspective of a new user, I wouldn't understand when the "default" happens, and what happens when the "default" doesn't happen. (TBH I find "Moderators may promote to Frontpage" confusing as well, but less so.)

I think I like the "This post may appear on the Frontpage" suggestion the best.